client/src/components/homepage.jsx

```diff
@@ -165,12 +177,13 @@ function mapStateToProps(state) {
   return {
     auth: state.auth,
     favoriteRecipe: state.recipeDetailReducer.favoriteRecipes,
-    upvotedRecipes: state.recipeReducer.upvotedRecipes
+    upvotedRecipes: state.recipeReducer.upvotedRecipes,
+    mostFavoriteRecipe: state.recipeReducer.mostFavorites.favoriteRecipes,
   };
 }
 
 Home.propTypes = {
   userSignupRequest: PropTypes.func.isRequired,
 };
 
-export default connect(mapStateToProps, { userSignupRequest, GetFavoriteRecipe, GetUpvotedRecipes })(Home);
+export default connect(mapStateToProps, { userSignupRequest, GetFavoriteRecipe, GetUpvotedRecipes, GetMostFavoriteRecipe })(Home);
```

Comment:
### Nitpick

Since the dispatches are getting much, don't you feel it's time you had a `mapDispatchToProps` function to handle this.